tools/fastlane-plugin/lib/fastlane/plugin/bugsnag/actions/bugsnag_notify_build.rb

```diff
@@ -0,0 +1,228 @@
+          FastlaneCore::ConfigItem.new(key: :app_version_code,
+                                       description: "Android app version code",
+                                       optional: true,
+                                       default_value: nil),
+          FastlaneCore::ConfigItem.new(key: :app_bundle_version,
+                                       description: "iOS/macOS/tvOS bundle version",
+                                       optional: true,
+                                       default_value: nil),
+          FastlaneCore::ConfigItem.new(key: :release_stage,
+                                       description: "Release stage being built, i.e. staging, production",
+                                       optional: true,
+                                       default_value: nil),
+          FastlaneCore::ConfigItem.new(key: :builder,
+                                       description: "The name of the entity triggering the build",
+                                       optional: true,
+                                       default_value: `whoami`),
+          FastlaneCore::ConfigItem.new(key: :repository,
+                                       description: "The git repository URL for this application",
+                                       optional: true,
+                                       default_value: nil),
+          FastlaneCore::ConfigItem.new(key: :revision,
+                                       description: "The source control revision id",
+                                       optional: true,
+                                       default_value: nil),
+          FastlaneCore::ConfigItem.new(key: :provider,
+                                       description: "The name of the source control provider, only required for on-premise services",
+                                       optional: true,
+                                       default_value: nil),
+          FastlaneCore::ConfigItem.new(key: :endpoint,
+                                       description: "Bugsnag deployment endpoint",
+                                       optional: true,
+                                       default_value: "https://build.bugsnag.com")
+        ]
+      end
+
+      private
+
+      def self.get_android_options(params)
+        require "xmlsimple"
+        if params[:config_file].nil?
+          config_file = Dir::glob("./src/main/AndroidManifest.xml").first
+          if config_file.nil?
+            puts "Not config_file option was provided, and your AndroidManifest could not be found. Bugsnag API key could not be automatically detected"
+            return
+          end
+        else
+          config_file = params[:config_file]
+        end
+
+        begin
+          config_hash = XmlSimple.xml_in(config_file)
+        rescue ArgumentError => e
+          raise ArgumentError.new "AndroidManifest.xml file not found, please point to your AndroidManifest file"
```

Comment:
The message here should indicate that the file could not be read. Fastlane also provides `UI.user_error!` for logging fatal errors.